crates/lib/src/cli.rs

```diff
@@ -295,6 +303,12 @@ pub(crate) enum InstallOpts {
 /// Subcommands which can be executed as part of a container build.
 #[derive(Debug, clap::Subcommand, PartialEq, Eq)]
 pub(crate) enum ContainerOpts {
+    /// Output JSON to stdout containing the container image metadata.
```

Comment:
Since this is a proposed stable API can you generate the initial manpages with `cargo xtask update-generated` and then populate it

Reply:
Done